lib/trento_web/plugs/app_jwt_auth_plug.ex

```diff
@@ -101,4 +99,25 @@ defmodule TrentoWeb.Plugs.AppJWTAuthPlug do
   @spec validate_refresh_token(binary()) :: {atom(), any()}
   defp validate_refresh_token(jwt_token),
     do: RefreshToken.verify_and_validate(jwt_token)
+
+  def user_allowed_to_renew?(%User{deleted_at: deleted_at, locked_at: locked_at})
```

Comment:
This function should come after `attach_refresh_token_to_conn`?
And are we sure it should be a public function?

Reply:
You're right, moved function and made it private